src/views/components/trace/trace-detail-chart-list.vue

```diff
@@ -281,6 +281,7 @@ export default {
         });
       });
       for (const i in segmentGroup) {
+        segmentGroup[i].children.sort((a, b) => a.startTime - b.startTime);
```

Comment:
Sorting all children is very dangerous, have do you know then are at the same level? Sharing the same parent?

Reply:
@TinyAllen Could you recheck this? What is the influence of this order?

Reply:
I am thinking about this, we have 4 spans
```
 ------s1-------s2--------s4
            +--------s3---------s5
```
Clearly, the start time order is s1, s2, s3, s4, s5, but in trace view, it should look like
```
s1
+ - s2
       + - s4
+ - s3
       + - s5
```

@TinyAllen Does the order of s3 and s4 affect the visualization of trace tree?

From my understanding, currently, the whole subtree is wrong, like this. 
```
s1
+ - s3
       + - s5
+ - s2
       + - s4
```
We can't guarantee the nodes of one subtree(s3--s5) are before another(s2--s4), especially in Java async span mode or Golang span. So we need to confirm if we keep the result as `s1-s2-s3-s4-s5`, the tree visualization is still right. This is my doubt.

Reply:
@wu-sheng In the example,
 ------s1-------s2--------s4
                +--------s3---------s5
if s3 is a async span（e.g. Hystrix）, the original order is s1, s2, s4, s3, s5
For PR，Sorting all children because the children of segmentGroup[i]  are at the same level，for example，segmentGroup[0].children = [s1, s2, s3, s4] ,segmentGroup[1].children = [s5] .

Reply:
In SkyWalking, there are two kinds of Async, you are talking about two segments, I am saying, in one segment, this could happen. Because SkyWalking has async span(not segment), and SkyWalking go SDK could do similar things too.